src/org/omegat/filters3/xml/xliff/EditXLIFFOptionsDialog.java

```diff
@@ -214,7 +234,7 @@ private void okButtonActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIR
 
     private void cancelButtonActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIRST:event_cancelButtonActionPerformed
         doClose(RET_CANCEL);
-    }//GEN-LAST:event_cancelButtonActionPerformed
+    }                                            
```

Comment:
This change looks wrong.

Reply:
I'm not sure why this removed because here is a part of auto generated code block....
I'd like to hand write here to revert.